src/components/network/network-explorer.js

```diff
@@ -125,6 +151,8 @@ const NetworkExplorer = ({
 
     const listeRef = useRef(null);
 
+    const maxLengthSubstationName = 21; // Max length of substation name
```

Comment:
Not sure if it's better to use css text ellipsis ? https://material-ui.com/system/display/#text-overflow
It makes it harder to detect which substations needs a tooltip. But then again, tooltips are not very userfriendly on touch devices

Reply:
Ha my bad, it already uses text ellipsis, and the limit of 21 chars is only used to decide to put the tooltip or not. it's ok then. We could use code like https://stackoverflow.com/questions/56588625/react-show-material-ui-tooltip-only-for-text-that-has-ellipsis to dynamically see if there is an ellipsis but since the drawer has a fixed width, the 21 chars is a good solution for now.

(until we decide that on 2000px large screen, we can afford to have a bigger drawer...need a UI expert for this)